xapi_db_load/backends/clickhouse_lake.py

```diff
@@ -39,34 +43,72 @@ def create_db(self):
         self.client.command(f"CREATE DATABASE IF NOT EXISTS {self.database}")
 
     def drop_tables(self):
+        self.client.command(f"DROP TABLE IF EXISTS {self.event_raw_table_name}")
         self.client.command(f"DROP TABLE IF EXISTS {self.event_table_name}")
+        self.client.command(f"DROP TABLE IF EXISTS {self.event_table_name_mv}")
         print("Tables dropped")
 
     def create_tables(self):
+        sql = f"""
+            CREATE TABLE IF NOT EXISTS {self.event_raw_table_name} (
+                event_id UUID NOT NULL,
+                emission_time DateTime64(6) NOT NULL,
+                event JSON NOT NULL,
+                event_str String NOT NULL,
+            )
+            ENGINE MergeTree ORDER BY (
+                emission_time,
+                event_id)
+            PRIMARY KEY (emission_time, event_id)
+        """
+        print(sql)
+        self.client.command(sql)
+
         sql = f"""
             CREATE TABLE IF NOT EXISTS {self.event_table_name} (
                 event_id UUID NOT NULL,
                 verb_id String NOT NULL,
                 actor_id UUID NOT NULL,
                 org String NOT NULL,
                 course_id String NOT NULL,
-                emission_time timestamp NOT NULL,
+                emission_time DateTime64(6) NOT NULL,
```

Comment:
This is actually a super important change. By default ClickHouse will round datetimes to the second, this allows it to store the timestamps in the same way that Python sends them and prevents issues with comparing times.